### PR TITLE
Provide Docs URLs in `--help` messaging

### PR DESCRIPTION
**Pull request by dacbd**
I did some very basic skimming of yargs to see if they had a better mechanism for this but didn't find anything (note  on the basic skimming,  I find their docs website difficult to read and search)

### CODE REVIEW

**Review by tasdomas — COMMENTED** [review]
I wonder if we need either a test or a cron action to validate that none of the links are dead.

**Comment by dacbd** [comment]
That should be easy enough to add

File: bin/cml.test.js [hunk 1]
```diff
@@ -31,3 +32,21 @@ describe('command-line interface tests', () => {
...
+    'workflow/rerun',
+    'tensorboard/connect',
+    'runner/launch',
+    'repo/prepare',
+    'pr/create',
+    'comment/create',
+    'comment/update',
+    'check/create',
+    'asset/publish'
+  ])('Check Docs Links', async (cmd) => {
```
Comment (dacbd) [inline bin/cml.test.js:47]:
```suggestion
  ])('Check Docs Link', async (cmd) => {
```

File: bin/cml.test.js [hunk 2]
```diff
@@ -31,3 +32,21 @@ describe('command-line interface tests', () => {
...
+  test.each([
+    'workflow/rerun',
+    'tensorboard/connect',
+    'runner/launch',
+    'repo/prepare',
+    'pr/create',
+    'comment/create',
+    'comment/update',
+    'check/create',
+    'asset/publish'
```
Comment (0x2b3bfa0) [inline bin/cml.test.js:46]:
```suggestion
    'workflow:rerun',
    'tensorboard:connect',
    'runner:launch',
    'repo:prepare',
    'pr:create',
    'comment:create',
    'comment:update',
    'check:create',
    'asset:publish'
```

For consistency with analytics?

**Review by tasdomas — APPROVED** [review]

File: bin/cml.test.js (hunk 2 above)
Comment (0x2b3bfa0) [reply bin/cml.test.js:46]: My bad, nevermind

Comment (casperdcl) [reply bin/cml.test.js:46]: going to be hard to keep this list updated as (sub)commands new are added.
